src/render/Texture.js

```diff
@@ -107,12 +127,43 @@ define([
          * @param {DrawContext} dc The current draw context.
          */
         Texture.prototype.bind = function (dc) {
-            dc.currentGlContext.bindTexture(dc.currentGlContext.TEXTURE_2D, this.textureId);
+            var gl = dc.currentGlContext;
+
+            gl.bindTexture(gl.TEXTURE_2D, this.textureId);
+
+            this.applyTexParameters(dc);
+
             dc.frameStatistics.incrementTextureLoadCount(1);
             return true;
         };
 
         /**
+         * Applies the configured texture parameters to the OpenGL context.
+         * @param {DrawContext} dc The current draw context.
+         */
+        Texture.prototype.applyTexParameters = function (dc) {
+            var gl = dc.currentGlContext;
+
+            // Configure the OpenGL texture magnification function. Use linear by default.
+            var textureMagFilter = this.texParameters[gl.TEXTURE_MAG_FILTER] || gl.LINEAR;
+            gl.texParameteri(gl.TEXTURE_2D, gl.TEXTURE_MAG_FILTER, textureMagFilter);
+
+            // Try to enable the anisotropic texture filtering only if we have a linear magnification filter.
+            // This can't be enabled all the time because Windows seems to ignore the TEXTURE_MAG_FILTER parameter when
+            // this extension is enabled.
+            if (textureMagFilter === gl.LINEAR) {
+                // Setup 4x anisotropic texture filtering when this feature is available.
+                // https://www.khronos.org/registry/webgl/extensions/EXT_texture_filter_anisotrop
+                var ext = (
+                    gl.getExtension("EXT_texture_filter_anisotropic") ||
```

Comment:
This used to be done once when creating the texture. Is this getExtension call expensive? Otherwise, we could still get it once in the constructor or even globally and simply use it here if present. What do you think?